### ISSUE TITLE
Frontend: Error when trying to load localhost:3000 in browser

### ISSUE THREAD

**Issue by dannymcgee** [postlight/headless-wp-starter#81]
I've gotten the WordPress backend up and running at localhost:8080. When I run `yarn install && yarn run dev` in the frontend directory, it looks successful:

```
yarn run v1.6.0
$ node server.js
> Using "webpack" config function defined in next.config.js.
DONE Compiled successfully in 2127ms

> Ready on http://localhost:3000
```

But when I try to visit localhost:3000 in the browser, I get an Internal Server Error, and this appears in the terminal:

```
{ Error: Cannot find module /mnt/c/users/danny/code/headless-wp-starter/frontend/.next/dist/pages/_error
    at _callee$ (/mnt/c/users/danny/code/headless-wp-starter/frontend/node_modules/next/dist/server/resolve.js:220:19)
    at tryCatch (/mnt/c/users/danny/code/headless-wp-starter/frontend/node_modules/next/node_modules/regenerator-runtime/runtime.js:65:40)
    at Generator.invoke [as _invoke] (/mnt/c/users/danny/code/headless-wp-starter/frontend/node_modules/next/node_modules/regenerator-runtime/runtime.js:299:22)
    at Generator.prototype.(anonymous function) [as next] (/mnt/c/users/danny/code/headless-wp-starter/frontend/node_modules/next/node_modules/regenerator-runtime/runtime.js:117:21)
    at step (/mnt/c/users/danny/code/headless-wp-starter/frontend/node_modules/next/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /mnt/c/users/danny/code/headless-wp-starter/frontend/node_modules/next/node_modules/babel-runtime/helpers/asyncToGenerator.js:28:13
    at <anonymous> code: 'ENOENT' }
```

That same exact error appears twice actually, I'm not sure if that's relevant. Have I done something horribly wrong? Let me know if you need anymore information. I'm on Windows 10 using a Linux subsystem.

**Comment by omattman** [comment]
Any news on this?

**Comment by omattman** [comment]
Hi @dannymcgee 

I managed to produce the correct display. I think this is the ongoing issue:

- Path to file is too long
or
- Next.js can't bundle it correctly, since i doesn't have access to the needed modules due to file path

To fix this:
1. Move up your directory to shorten pathname. I changed mine from `/mnt/c/users/matthias-pc/dev/headless-wp-starter` to `/mnt/c/dev/headless-wp-starter`
2. Follow the installation guide as instructed in the [`README.md`](https://github.com/postlight/headless-wp-starter/blob/master/README.md)
